### DISCUSSION TITLE
Nice!

### DISCUSSION THREAD

**Discussion by meinja** [mfncooper/paracon#2] [General]
This is a welcome addition.  I tried it here for a couple minutes on my Linux Mint machine.  Easy to run, no installion necessary.  I did need to make the file executable.  I've been looking for a basic packet terminal interface for Linux that can run alongside my node and use Direwolf's AGW interface, and this fits the bill :)
73
Jeff KP3FT

**Comment by mfncooper** [comment]
Thanks for the feedback!

Unfortunately, the only way to have the file be executable when downloaded would be to package it inside something else, which would then require an additional installation step. I've updated the documentation to reflect, as you said, that you need to mark the file executable yourself. Sorry about that.

I've also just released version 1.1.0, with some minor updates.
